omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java

```diff
@@ -90,10 +90,7 @@ public void controller(FragmentConfiguration config,
     	CollectionUtils.filter(ret, new Predicate<Extension>() {
     		@Override
     		public boolean evaluate(Extension extension) {
-       		 	if(extensionPointId.equals(extension.getExtensionPointId())) {
-	              return true;
-	            }
-	            return false;
+       		 	return extensionPointId.equals(extension.getExtensionPointId());
 	        }
 	    });
     	return ret; 
```

Comment:
I checked the whitespace and I can see quite a chaos here:
![image](https://user-images.githubusercontent.com/9057882/99976543-dc11b800-2da3-11eb-86ab-bf31192b280d.png)
(dots are spaces and arrows are tabs)

It's not your fault, it's the fault of the person who wrote it, but you have a chance to improve it.

I propose such a solution:
```suggestion
    			return extensionPointId.equals(extension.getExtensionPointId());
    		}
    	});
    	return ret; 
```
What will look like this:
![image](https://user-images.githubusercontent.com/9057882/99977254-b0430200-2da4-11eb-8034-528fafa3ead8.png)
It's still not the best style of writing, but at least it is uniform for this section of the file.

Of course, you can also suggest your own corrections. ;)

Reply:
Setting the dev environment as per the community priority at https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse This makes the codebase to be formatted on "save action" as per OpenMRSFormatter.xml file which is imported into the IDE. 
This is likely the reason why such spaces exists. I stand to be corrected

Reply:
Hey @sacull ! Any one is free to take up this ticket!

Reply:
Sorry, but I didn't have access to Github for a month.

> Hey @sacull I think such spaces are as a result of setting the dev environment as per the community priority at https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse This makes the codebase to be formatted on "save action" as per OpenMRSFormatter.xml file which is imported into the IDE.
> This is likely the reason why such spaces exists. I stand to be corrected
In this case, the IDE would only make spaces or only tabs, not sometimes this and sometimes this. I think the reason may be different.

> Hey @sacull ! Any one is free to take up this ticket!
In my opinion, it would be best if you finished this ticket. However, if you prefer to abandon it, no one will take it, if you are assigned to it and the PR is open. ;)